codiga/graphql/common.py

```diff
@@ -19,7 +20,7 @@ def do_graphql_query(api_token, payload):
     :param payload: the payload we want to send.
     :return: the returned JSON object
     """
-    headers = {API_TOKEN_HEADER: api_token}
+    headers = {API_TOKEN_HEADER: api_token, USER_AGENT_HEADER: USER_AGENT_CLI}
```

Comment:
All headers for the user agent should be `Name/Version`. You can find the version of the cli tool in `https://github.com/codiga/clitool/blob/master/codiga/version.py`